warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java

```diff
@@ -119,6 +119,14 @@ public Entry<Key,Document> apply(@Nullable Entry<Key,Document> keyDocumentEntry)
             this.countingMap = groupingInfo.getCountsMap();
```

Comment:
Up above we may want to check for a FinalDocument before adding it into the fold.  See the UniqueTransform for the code into apply method.

Reply:
Done